AardvarkMailUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name     = 'AardvarkMailUI'
-  s.version  = '1.0.0'
+  s.version  = '1.1'
```

Comment:
Let's make this `1.1.0` to stay consistent

Reply:
Caught that too!